test/Microsoft.Framework.FileSystemGlobbing.Tests/FunctionalTests.cs

```diff
@@ -29,6 +29,17 @@ public void Dispose()
         }
 
         [Fact]
+        public void DuplicatePatterns()
```

Comment:
There are tests added to test slash and backslash in the input patterns. Those tests are added when tests on *nix was fixed.